interp/cfg.go

```diff
@@ -960,7 +960,14 @@ func (interp *Interpreter) cfg(root *node, sc *scope, importPath, pkgName string
 			}
 			wireChild(n)
 
-		case declStmt, exprStmt, sendStmt:
+		case sendStmt:
+			if !isChan(n.child[0].typ) {
+				err = n.cfgErrorf("cannot send to non-channel %s", n.child[0].typ.id())
```

Comment:
Shouldn't we prefix the message with "invalid operation: ", to make it match the go compiler?

```
package main

import "math"

func main() {
	math.NaN() <- 4
}
```

```
% go run /Users/mpl/main.go 
# command-line-arguments
/Users/mpl/main.go:6:2: invalid operation: cannot send to non-channel math.NaN() (value of type float64)
% 
% yaegi run /Users/mpl/main.go 
run: /Users/mpl/main.go:6:2: cannot send to non-channel float64
```

Reply:
Yes, it makes sense